src/transformState.js

```diff
@@ -4,8 +4,39 @@
  * @param {Object} state
  * @param {Object[]} actions
  */
+const removeKey = (action, data) => {
+  const { keysToRemove } = action;
```

Comment:
Why do you need the whole action object inside the function when you only need the `keysToRemove` array ?

Reply:
using ' const { type, extraData, keysToRemove } = action;' destructuring and using keysToRemove in the loop